src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -168,17 +173,13 @@ foam.CLASS({
             onKey: true
           })
         .end()
-        .add(this.slot(function (
-          data$wizardlets,
-          data$wizardPosition,
-          data$availabilityInvalidate
```

Comment:
The `availabilityInvalidate` property doesn't exist anymore. It would be nice to know when it was removed but I can't think of a convenient way to find out.

Reply:
I was getting a console error when I was debugging and realized we don't use it anywhere so I removed it - no idea how to track it down though...